docker-compose.yml

```diff
@@ -2,8 +2,18 @@ version: '3'
 
 services:
   algod:
+    image: wwinder/algod:latest
+    ports:
+      - "4001:8080" # algod
+      - "4160:4160" # gossip
+      - "9100:9100" # prometheus
+    environment:
+      - DEV_MODE
+      - TOKEN=aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
+
+  # TODO: do not merge with this kmd service. This is only needed cause I don't want to
+  # stand-up a seperate service locally
+  kmd:
     image: makerxau/algorand-sandbox-dev:latest
```

Comment:
Does this even work? I would think you want kmd to talk /w algod. It could be added to our docker container.

Reply:
Didn't run the full test on it so it may not work (and it was only temporary). But yeah, adding KMD to the docker would be great.

Reply:
to clarify - current pyteal tests are dry run only so not using KMD